packages/internals/src/engine-commands/getEngineVersion.ts

```diff
@@ -8,11 +7,9 @@ import execa from 'execa'
 import { resolveBinary } from '../resolveBinary'
 import { load } from '../utils/load'
 
-const debug = Debug('prisma:getVersion')
```

Comment:
Intentional removal of debug logging?

Reply:
Looks like it wasn't used (so no logs were emitted here)